dataprofiler/tests/profilers/test_categorical_column_profile.py

```diff
@@ -216,6 +231,17 @@ def test_categorical_merge(self):
                          "NaN": 1, "None": 1, "nan": 1, "null": 1}
         self.assertDictEqual(expected_dict, profile3._categories)
 
+        report = profile3.profile
+        report.pop("times")
+        expected_profile = dict(
+            categorical=False,
```

Comment:
I'm confused, a lot of these test_categorical functions have categorical as false in the expected dict. Is that intended? It seems like either is_match isn't accurate or these test cases aren't representative.

Reply:
I think the intent is that we are merging and to prove that even if a categorical profiler doesn't start as categorical, it can update to one as it gets more data and the report back is updating as it should based on whether it is categorical or not.